### ISSUE TITLE
Unhandled error during execution of unmounted hook in nuxt3

### ISSUE THREAD

**Issue by chupacabramiamor** [n1crack/vuefinder#72]
when i trying to unshow the VueFinder component, I'm faced with this error

![image](https://github.com/user-attachments/assets/98c7d4ee-80d3-4406-ba20-a91b2b01db90)

Code sample:
```
<template>
  <PageHeader title="Website uploads"></PageHeader>

  <PageBody>
    <a href="#" @click.prevent="show = false">Hide</a>
    <VueFinder :request="`${runtimeConfig.public.appUrl}/api/crm/uploads`" v-if="show"/>
  </PageBody>
</template>

<script setup>
definePageMeta({
  middleware: 'auth',
  nav: 'uploads',
});

const runtimeConfig = useRuntimeConfig();

const show = ref(true);
</script>
```

This behavior is watching only in dev-mode in Nuxt3. Generated version of App works fine

**Comment by n1crack** [comment]
It should be fixed, can you try again ?

**Comment by ximepa** [comment]
> It should be fixed #73, can you try again ?

works perfect. Thx!
